source/patterns/@aws-solutions-constructs/core/lib/input-validation.ts

```diff
@@ -71,6 +71,9 @@ export interface VerifiedProps {
 export function CheckProps(propsObject: VerifiedProps | any) {
   let errorMessages = '';
   let errorFound = false;
+  const isQueueFifo: boolean = propsObject?.queueProps?.fifo;
```

Comment:
There's no rule that these must be near the top of the routine. Their only purpose is to introduce names that add clarity to the decisions in line 108 - go ahead and put them just above that decision.